### PR TITLE
RHIDP-4248 Finish removing authentication sections from GSG 

### PR DESCRIPTION
**Pull request by themr0c**
Finish removing authentication sections from the Getting started guide.

Only the `integration` section was missing in the Authentication guide.

All the rest has already been updated in the Authentication guide.

**IMPORTANT: Do Not Merge - To be merged by Docs Team Only**

**Version(s):** 1.3.1 

**Issue:** https://issues.redhat.com/browse/RHIDP-4248

**Link to docs preview:**

**Reviews:**
- [ ] SME: @ mention assignee
- [ ] QE: @ mention assignee
- [ ] Docs review: @ mention assignee
- [ ] Additional review: @mention assignee (by writer)

**Additional information:**

### CODE REVIEW

File: titles/getting-started-rhdh/title-getting-started.adoc [hunk 1]
```diff
@@ -19,7 +19,6 @@ include::modules/installation/ref-rhdh-sizing.adoc[leveloffset=+1]
 // supported configs and customization
 include::modules/getting-started/ref-rhdh-supported-configs.adoc[leveloffset=+1]
 include::assemblies/assembly-add-custom-app-file-openshift.adoc[leveloffset=+2]
-include::modules/getting-started/proc-add-source-control-rhdh-catalog.adoc[leveloffset=+2]
```
Comment (hmanwani-rh) [inline titles/getting-started-rhdh/title-getting-started.adoc:22]: In the [Bulk Import PR](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/616), wasn't the Bulk Import section added under Source Control? I noticed that in this PR, we're removing it. Just trying to clarify as it seems the structure and preview haven't been generated.

**Review by albarbaro — APPROVED** [review]
looks good to me. thanks!

File: titles/getting-started-rhdh/title-getting-started.adoc (hunk 1 above)
Comment (themr0c) [reply titles/getting-started-rhdh/title-getting-started.adoc:22]: Indeed, I updated this other PR to reflect the change.

File: assemblies/assembly-authenticating-with-github.adoc [hunk 2]
```diff
@@ -11,3 +11,9 @@ include::modules/authentication/proc-enabling-authentication-with-github.adoc[le
 
 include::modules/authentication/proc-provisioning-users-from-github-to-the-software-catalog.adoc[leveloffset=+1]
 
+
+include::modules/authentication/proc-configuring-github-integration.adoc[leveloffset=+1]
+
+
+include::modules/authentication/proc-enabling-github-discovery.adoc[leveloffset=+1]
+
```
Comment (themr0c) [inline assemblies/assembly-authenticating-with-github.adoc:19]:
```suggestion

```

File: assemblies/assembly-authenticating-with-github.adoc [hunk 3]
```diff
@@ -10,4 +10,3 @@ include::modules/authentication/proc-enabling-authentication-with-github.adoc[le
 
 
 include::modules/authentication/proc-provisioning-users-from-github-to-the-software-catalog.adoc[leveloffset=+1]
```
Comment (themr0c) [inline assemblies/assembly-authenticating-with-github.adoc:12]:
```suggestion
include::modules/authentication/proc-provisioning-users-from-github-to-the-software-catalog.adoc[leveloffset=+1]

```

**Review by PatAKnight — APPROVED** [review]
Looks good to me as well

**Comment by themr0c** [comment]
@hmanwani-rh Good to merge.

**Comment by hmanwani-rh** [comment]
@themr0c Please cherry-pick to release-1.3 branch manually

**Pull request merged**